.travis.yml

```diff
@@ -1,7 +1,7 @@
 dist: trusty
 sudo: false
 language: node_js
-node_js: 8.8.0
+node_js: 8.9.0
 before_install:
 - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.2.1
 - export PATH=$HOME/.yarn/bin:$PATH
```

Comment:
Need to have this:

```yaml
before_script:
- export DISPLAY=':99.0'
- sh -e /etc/init.d/xvfb start
```

otherwise browser tests would hang in CI